src/main/java/core/basesyntax/util/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(List<String> content, String filePath) {
+        try {
+            Files.write(Path.of(filePath), content);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write file by path " + filePath);
```

Comment:
Consider including the original exception `e` in the `RuntimeException` to provide more context about the error. This can be done by using `throw new RuntimeException("Cannot write file by path " + filePath, e);`.